### ISSUE TITLE
Where does Unsaved Original Data Streams go?

### ISSUE THREAD

**Issue by svk27** [openobserve/openobserve#5081]
### Discussed in https://github.com/openobserve/openobserve/discussions/5079

<div type='discussions-op-text'>

<sup>Originally posted by **svk27** November 16, 2024</sup>
Hey everyone, I'm just trying out open observe, and it is definitely something!
What I don't understand is., where does the log/stream data get's stored at when the "Store Original Data" is set to off.

For example, the below streams has a similar number of rows, except, one has "Store Original Data" set to "on".
![Screenshot 2024-11-16 214912](https://github.com/user-attachments/assets/2461fa4d-fff8-4695-bc74-8293b04e3c9a)

**For more context:**
I'm using Easypanel to host and manage the Docker container, with no volume/storage mounts attached, 
using AWS S3 and postgres for HA.

So basically, if I were to redeploy the container, it should purge everything, but it is not doing that, so I am not sure.

The .env contents being:
ZO_ROOT_USER_EMAIL="xxx"
ZO_ROOT_USER_PASSWORD="xxx"
LOCAL_MODE="false"
ZO_LOCAL_MODE_STORAGE="s3"
ZO_S3_REGION_NAME="eu-central-1"
ZO_S3_ACCESS_KEY="xxx"
ZO_S3_SECRET_KEY="xxx"
ZO_S3_BUCKET_NAME="xxx"
ZO_S3_PROVIDER="s3"
ZO_META_STORE="postgres"
ZO_META_POSTGRES_DSN="postgres://postgresxxx@xxx:123/p1"

**Comment by hengfeiyang** [comment]
First:

1. metadata is in Postgres, it means the metadata(stream, stats, users) still in database.
2. stored original data in s3. reinstall will not remove the data in s3.

Second:
`Store Original Data` just a option if store original format data, if enable it will store the original data with keep the format in an additional field. otherwise we will flatten the data structure and didn't keep the original structure.

**Comment by svk27** [comment]
Good insight, thank you for the same.

So, do you still think that Records, Ingested data, etc where Store Original Data set to false shows "0", still a bug, 
or it is what it should store?
